zengo/service.py

```diff
@@ -119,9 +119,10 @@ def create_remote_zd_user_for_local_user(self, local_user):
             print(a.response.json())
             details = a.response.json()["details"]
             if any([d[0]["error"] == "DuplicateValue" for d in details.values()]):
-                remote_zd_user, is_definite_match = self.get_remote_zd_user_for_local_user(
-                    local_user
-                )
+                (
```

Comment:
This was Black, not me.

Reply:
Fair enough - I guess `black` has changed its output slightly as they bump versions